scripts/create_datacheck_tickets.pl

```diff
@@ -0,0 +1,172 @@
+
+=item B<-r[elease]> <release>
+
+Mandatory. Ensembl release version.
+
+=item B<-p[roject]> <JIRA_project>
+
+Mandatory. JIRA project name, e.g. 'ENSPROD'.
+
+=item B<-user> <JIRA_user>
+
+Optional. JIRA username. If not given, uses environment variable $USER 
+as default.
+
+=item B<-components> <>
+
+Optional. JIRA component tags. Multiple values can be given as separate 
+-components parameters, or as a single comma-separated string.
+
+=item B<-dry_run>, B<-dry-run>
+
+In dry-run mode, the JIRA tickets will not be submitted to the JIRA
+server. By default, dry-run mode is off.
+
+=item B<-h[elp]>
+
+Print usage information.
+
+=back
+
+=cut
+
+
+use warnings;
+use strict;
+
+use Cwd 'abs_path';
+use File::Basename;
+use Getopt::Long;
+use Pod::Usage;
+use POSIX;
+
+use Bio::EnsEMBL::Compara::Utils::JIRA;
+
+use Data::Dumper;
+
+my ( $dc_file, $release, $project, $user, @components, $dry_run, $help );
+$dry_run = 0;
+$help    = 0;
+GetOptions(
+    "i|input=s"       => \$dc_file,
+    "r|release=s"     => \$release,
+    "p|project=s"     => \$project,
+    "user=s",         => \$user,
+    "components:s"    => \@components,
+    'dry_run|dry-run' => \$dry_run,
+    "h|help"          => \$help,
+);
+pod2usage(1) if $help;
+die "Cannot find $dc_file - file does not exist" unless -e $dc_file;
+die pod2usage(1) if (! ($dc_file && $release && $project));
+# Get file absolute path and basename
+my $dc_abs_path = abs_path($dc_file);
+my $dc_basename = fileparse($dc_abs_path, qr{\.[a-zA-Z0-9_]+$});
+# Get timestamp that will be included in the summary of each JIRA ticket
+my $timestamp = strftime("%d-%m-%Y %H:%M:%S", localtime time);
+# Get a new Utils::JIRA object to create the tickets for the given division and
+# release
+my $jira_adaptor = Bio::EnsEMBL::Compara::Utils::JIRA->new(
+    -USER     => $user,
+    -DIVISION => '',  # defined to avoid using $ENV{'COMPARA_DIV'} (default)
+    -RELEASE  => $release,
+    -PROJECT  => $project
+);
+# Parse Datacheck information from input TAP file
+my $testcase_failures = parse_datachecks($dc_file, $timestamp);
+# Create initial ticket for datacheck run - failures will become subtasks of this
+my $dc_task_json_ticket = [{
+    assignee    => $jira_adaptor->{_user},
+    summary     => "$dc_basename ($timestamp)",
+    description => "Datacheck failures raised on $timestamp\nFrom file: $dc_abs_path",
+}];
+# Create subtask tickets for each datacheck failure
+my @json_subtasks;
+foreach my $testcase ( keys %$testcase_failures ) {
+    my $failure_subtask_json = {
+        summary     => $testcase,
+        description => $testcase_failures->{$testcase},
+    };
+    push(@json_subtasks, $failure_subtask_json);
+}
+# Add subtasks to the initial ticket
+$dc_task_json_ticket->[0]->{subtasks} = \@json_subtasks;
+# Create all JIRA tickets
+my $dc_task_keys = $jira_adaptor->create_tickets(
+    -JSON_OBJ         => $dc_task_json_ticket,
+    -DEFAULT_PRIORITY => 'Blocker',
```

Comment:
It'd be useful if this was a script parameter, with this as the default value (since some datachecks might be advisory, and therefore not necessarily blocking).